experimental/design-notes.txt

```diff
@@ -0,0 +1,112 @@
+I started comparing OpenTelemetry metrics to other metric APIs and wanted to jot down observations and questions that came up along the way:
+
+Micrometer (MM) - https://micrometer.io/docs/concepts
+
+1. Do the names that OpenTelemetry chose match industry conventions? 
+For the MM library there are different types of Meters such as timers, counters, gauges, etc. Depending on their type, meters implement operations such
```

Comment:
I think I heard may calls for a "timing" type counter.  Maybe we include a TimeRange counter of the form...

using (var timer = new Timer(...))
{
    // Do Operations...
}